### PR TITLE
CR-1054524 XRT - Remove the five second delay workaround for CMC/SC b…

### PR DESCRIPTION
**Pull request by xdavidz**
I am still testing it. Will update test report when testing is done.

### CODE REVIEW

**Comment by xdavidz** [comment]
@AShivangi @houlz0507 Got clear definition for checking stable SC via CMC. Could you please help me review if this change is appropriate?

File: src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp [hunk 1]
```diff
@@ -500,15 +503,48 @@ int XMC_Flasher::writeReg(unsigned RegOffset, unsigned value) {
...
+{
+    auto entry = map.find(status);
+
+    return entry != map.end() ?
+        "0x" + std::to_string(entry->first) + "(" + entry->second + ")":
+	"0x" + std::to_string(entry->first);
+}
+
+/* XMC means CMC */
+void XMC_Flasher::initCMCStatusMap()
```
Comment (AShivangi) [inline src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp:516]: Maybe make this an enum?

Comment (xdavidz) [reply src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp:516]: We need to print the string like "NOT READY" or "READY". You mean we convert the 0,1,2,4 to enum?

Comment (AShivangi) [reply src/runtime_src/core/pcie/tools/xbmgmt/xmc.cpp:516]: I see. This looks good.

**Comment by keryell** [comment]
There should not be a mix of spaces and tabs in the code.

**Comment by xdavidz** [comment]
> There should not be a mix of spaces and tabs in the code.

C in kernel uses 8 character as one single tab, CPP in userland uses 4 spaces.
Please let me know which line(s) is(are) broken, I will fix them.

**Pull request merged**